pontoon/base/tests/models/test_translation.py

```diff
@@ -273,3 +273,23 @@ def test_translation_not_in_terminology_saved(
     TranslationFactory.create(locale=locale_a, entity=entity_a)
 
     assert not reset_term_translation_mock.called
+
+
+@pytest.mark.django_db
+def test_machinery_sources_values(locale_a, entity_a):
+    """
+    Return comma-separated machinery_sources values for the given translation.
+    """
+    translation_no_machinery_sources = TranslationFactory.create(
+        locale=locale_a, entity=entity_a,
+    )
+    assert translation_no_machinery_sources.machinery_sources_values == ""
+
+    translation = TranslationFactory.create(
+        locale=locale_a,
+        entity=entity_a,
+        machinery_sources=["translation-memory", "google-translate"],
```

Comment:
You could replace this with `Translation.MachinerySource.TranslationMemory/GoogleTranslate`.